src/App.test.jsx

```diff
@@ -1,8 +1,23 @@
+
+/* eslint-disable import/extensions */
 import React from 'react';
-import ReactDOM from 'react-dom';
-import App from './App';
+import toJson from 'enzyme-to-json';
+import { shallow } from 'enzyme';
+import configureStore from 'redux-mock-store';
+import thunkMiddleware from 'redux-thunk';
+import { Provider } from 'react-redux';
+import SHEET_API from './utils/Api';
+import App from './App.jsx';
 
-it('renders without crashing', () => {
-  const div = document.createElement('div');
-  ReactDOM.render(<App />, div);
+const middleware = [thunkMiddleware.withExtraArgument({ SHEET_API })];
+const mockStore = configureStore(middleware);
+describe('App component', () => {
+  it('should render a component with props as specified ', () => {
+    const component = shallow(
+      <Provider store={mockStore({})}>
```

Comment:
@HamidAghdaee I know we haven't (in our work here at G) tended to shallow test our components as they are wrapped in higher order components, what do you think?

Reply:
@Gwillison415 I agree with @joshwyatt no need to test a connected component.